api/groups/nodeGroup_test.go

```diff
@@ -60,6 +61,13 @@ type queryResponse struct {
 	Result []string `json:"result"`
 }
 
+type epochStartResponse struct {
+	Data struct {
```

Comment:
Oh, inline structs - I didn't know they were possible.

Reply:
yes, it's easier for testing, but for production code is still better to have separated structures